site/tutorials/keptn-litmus-018.md

```diff
@@ -0,0 +1,378 @@
+
+Let us add the `experiment.yaml` file that holds the chaos experiment instructions. It will be picked up by the LitmusChaos integration of Keptn each time a test is triggered. Therefore, Keptn makes sure that both, JMeter tests as well as LitmusChaos tests, are executed during the `test` task sequence.
+
+```
+keptn add-resource --project=litmus --stage=chaos --service=helloservice --resource=./litmus/experiment.yaml --resourceUri=litmus/experiment.yaml
+```
+
+Great job - the file is added and we can move on!
+
+## Configure Prometheus
+Duration: 2:00
+
+Before we are going to run the experiment, we have to make sure that we have some observability software in place that will actually monitor how the service is behaving under the testing conditions. 
+
+1. Let's use the Keptn CLI to configure Prometheus. It will set up a Prometheus deployment and configures it to be ready for Keptn usage. 
+
+    ```
+    keptn configure monitoring prometheus --project=litmus --service=helloservice
+    ```
+
+1. Next, we are going to add a `blackbox-exporter` for Prometheus that is able to observe our service under test from the outside, i.e., as a blackbox. 
+
+    ```
+    kubectl apply -f ./prometheus/blackbox-exporter.yaml
+    kubectl apply -f ./prometheus/prometheus-server-conf-cm.yaml -n monitoring
+    ```
+
+1. Finally, restart Prometheus to pick up the new configuration
+    
+    ```
+    kubectl delete pod -l component=server -n monitoring
+    ```
+
+Now everything is in place, let's run our experiments and evaluate the resilience of our demo application!
+
+## Run experiment
+Duration: 4:00
+
+We are now ready to kick off a new deployment of our test application with Keptn and have it deployed, tested, and evaluated.
+
+1. Let us now trigger the deployment, tests, and evaluation of our demo application.
+
+    ```
+    keptn trigger delivery --project=litmus --service=helloservice --image=jetzlstorfer/hello-server:v0.1.1
+    ```
+
+1. Let's have a look in the Keptn bridge what is actually going on. We can use this helper command to retrieve the URL of our Keptn bridge.
+
+    ```
+    echo http://$(kubectl -n keptn get ingress api-keptn-ingress -ojsonpath='{.spec.rules[0].host}')/bridge
+    ```
+
+    The credentials can be retrieved via the following commands:
+
+    <!-- command -->
+    ```
+    echo Username: $(kubectl get secret -n keptn bridge-credentials -o jsonpath="{.data.BASIC_AUTH_USERNAME}" | base64 --decode)
+    echo Password: $(kubectl get secret -n keptn bridge-credentials -o jsonpath="{.data.BASIC_AUTH_PASSWORD}" | base64 --decode)
+    ```
+
+    ![](./assets/keptn-litmus/litmus-first-run-started.png)
+
+1. We can see that the evaluation failed, but why is that?
+
+    ![](./assets/keptn-litmus/litmus-first-run.png)
+
+1. Let's take a look at the evaluation - lick on the *chart* icon in the red evaluation tile.
```

Comment:
```suggestion
1. Let's take a look at the evaluation - click on the *chart* icon in the red evaluation tile.
```
"lick on the icon" may be my favorite typo of the day!  ;-)

Reply:
yes, it was there in the older version of [tutorials](https://tutorials.keptn.sh/tutorials/keptn-litmus-011/index.html?index=..%2F..index#16) as well